parallel-crypto3/benchmarks/polynomial_dfs_benchmark.cpp

```diff
@@ -0,0 +1,215 @@
+//---------------------------------------------------------------------------//
```

Comment:
Was this change intentional?

Reply:
Yes, I moved the tests to match the crypto3 layout. It also has benchmarks on the top level